pkgs/servers/tt-rss/default.nix

```
@@ -1,16 +1,17 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchgit }:
 
 stdenv.mkDerivation rec {
   pname = "tt-rss";
-  year = "21";
-  month = "06";
-  day = "21";
+  year = "22";
+  month = "08";
+  day = "01";
   version = "20${year}-${month}-${day}";
```

```suggestion
  version = "2022-08-01";
```
and update installPhase. Also this should probably be prefixed with ``unstable-``.